dashboard/test/models/lesson_test.rb

```diff
@@ -215,6 +215,34 @@ class LessonTest < ActiveSupport::TestCase
     assert_equal script.summarize_for_lesson_show, summary[:unit]
   end
 
+  test 'lesson edit summary does not preprocess markdown' do
+    lesson = create :lesson, lesson_group: create(:lesson_group)
+    Services::MarkdownPreprocessor.expects(:process).never
+    lesson.summarize_for_lesson_edit
+  end
```

Comment:
better to `expects(:process!)` here, to prevent calls to both `.process` and `.process!`?